src/sync_template.sh

```diff
@@ -36,14 +36,19 @@ NEW_BRANCH="${PR_BRANCH_NAME_PREFIX}_${NEW_TEMPLATE_GIT_HASH}"
 
 echo "::group::Check new changes"
 echo "::debug::new Git HASH ${NEW_TEMPLATE_GIT_HASH}"
-if [ -r ${TEMPLATE_VERSION_FILE_NAME} ]
-then
+
+if [[ -f "$TEMPLATE_VERSION_FILE_NAME" ]]; then
+  echo "::debug::version file is located in root folder"
+else
+  echo "::debug::version file is located either in .github folder or not present"
+  TEMPLATE_VERSION_FILE_NAME=".github/$TEMPLATE_VERSION_FILE_NAME"
```

Comment:
same like below, small naming issue. With the directory it is not a `FILE_NAME` anymore. Pls check the notes below as a discussion item.